packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -824,6 +848,117 @@ class ThemeData extends Diagnosticable {
     properties.add(new DiagnosticsProperty<IconThemeData>('accentIconTheme', accentIconTheme));
     properties.add(new DiagnosticsProperty<SliderThemeData>('sliderTheme', sliderTheme));
     properties.add(new DiagnosticsProperty<ChipThemeData>('chipTheme', chipTheme));
+    properties.add(new DiagnosticsProperty<AdaptiveWidgetThemeData>('adaptiveWidgetTheme', adaptiveWidgetTheme));
+  }
+}
+
+/// Describes whether descendent Material Design widgets of should automatically
+/// adapt to use Cupertino widgets when running on iOS.
+///
+/// A [Map] from Material widget types to a boolean describes whether the
+/// adaptation should be made for widgets that is wrapped with [PlatformBuilder]
+/// descendant to this [Theme] and whose [PlatformBuilder.themeAdaptiveType]
+/// matches the type in the [adaptiveWidgets] [Map].
+///
+/// When a widget type is not explicitly specified in [adaptiveWidgets], the
+/// [defaultAdaptiveness] value is used.
+///
+/// The singleton [none] instance can be used to signal no adaptation on any
+/// descendent widgets.
+///
+/// The singleton [bundled] instance can be used to signal adaptation on all
+/// Material widgets.
```

Comment:
presumably not all, but those that are specified to do so?

Reply:
ah ya, that's misleading. Corrected